features/base/tracks/actionTypes.js

```diff
@@ -1,3 +1,4 @@
 export const TRACK_ADDED = 'TRACK_ADDED';
+export const TRACK_MUTE_CHANGED = 'TRACK_MUTE_CHANGED';
```

Comment:
Actions are objects in which the type property is required. What's obvious to you is the type property only and the other properties of the action of that type must be documented in the action type JavaScript documentation.